src/runtime_config/gov.lua

```diff
@@ -98,4 +98,11 @@ return {
   end
   return true]],
 
+  update_ca_cert = [[
```

Comment:
Should this be added to all governance sample files?

Reply:
Theoretically yes, but I will rip this out in the next PR.